.github/workflows/maestro_test.yml

```
@@ -0,0 +1,147 @@
+
+      - name: Cache node_modules
+        id: cache-node-modules
+        uses: actions/cache@v4
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-node_modules-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-node_modules-
+
+      - name: Install Dependencies
+        run: yarn install 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)